src/libraries/System.Drawing.Common/src/Interop/Windows/Interop.Comdlg32.cs

```diff
@@ -15,7 +15,7 @@ internal static partial class Comdlg32
         internal static extern bool PrintDlg([In, Out] PRINTDLGX86 lppd);
```

Comment:
I think you also need to change this argument to `ref PRINTDLGX86 lppd` (and same for the !X86 version) to compensate for the class -> struct switch. I am curious whether this problem is going to be caught by tests...

Reply:
I was catched by the tests. Question. I see `new HandleRef(data, handle)` near usages of  `PRINTDLGX86 data`. I assume that I should get rid of `HandleRef` since they are not needed anymore. and just use `handle` itself.

Reply:
Agree.